usaspending_api/agency/v2/views/agency_base.py

```diff
@@ -81,3 +81,18 @@ def pagination(self):
     @property
     def filter(self):
         return self.request.query_params.get("filter")
+
+
+class PaginationMixin:
```

Comment:
This new class looks too similar to `ListMixin`. Could `ListMixin` be replaced by `PaginationMixin` instead of nearly duplicating the code? The biggest differences appear to be leveraging class properties of `self.sortable_columns` & `self.default_sort_column`